trade-route-example-config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "goerli": {
+    "type": "collateral",
+    "token": "0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6",
+    "owner": "0x5bA371aeA18734Cb7195650aFdfCa4f9251aa513",
+    "mailbox": "0xCC737a94FecaeC165AbCf12dED095BB13F037685",
+    "interchainGasPaymaster": "0xf857706CE59Cb7AE6df81Bbd0B0a656dB3e6beDA"
+  },
+  "alfajores": {
+    "type": "synthetic",
+    "name": "Dai",
+    "symbol": "DAI",
+    "totalSupply": 0,
+    "token": "0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6",
+    "owner": "0x5bA371aeA18734Cb7195650aFdfCa4f9251aa513",
+    "mailbox": "0xCC737a94FecaeC165AbCf12dED095BB13F037685",
+    "interchainGasPaymaster": "0xf857706CE59Cb7AE6df81Bbd0B0a656dB3e6beDA"
+  },
+  "fuji": {
+    "type": "synthetic",
+    "name": "Dai",
+    "symbol": "DAI",
+    "totalSupply": 0,
+    "token": "0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6",
+    "owner": "0x5bA371aeA18734Cb7195650aFdfCa4f9251aa513",
+    "mailbox": "0xCC737a94FecaeC165AbCf12dED095BB13F037685",
+    "interchainGasPaymaster": "0xf857706CE59Cb7AE6df81Bbd0B0a656dB3e6beDA"
+  },
```

Comment:
Seems like these configs could be DRY-ed up with other remote configs but nbd

Reply:
Also was similar before, but again wanted to move to a dependency-less format